.travis.yml

```diff
@@ -25,8 +25,15 @@ jobs:
         - tests/scripts/all.sh -k build_arm_linux_gnueabi_gcc_arm5vte build_arm_none_eabi_gcc_m0plus
 
     - name: full configuration
+      os: linux
+      dist: focal
+      addons:
+        apt:
+          packages:
+          - clang
```

Comment:
Travis has clang installed by default, but it's Clang 7. Installing the `clang` package explicitly doesn't help, even though that brings version 10. I think the problem is that Travis installs `/usr/local/bin/clang` which is Clang 7 and `all.sh` calls `clang` from `$PATH` which ends up being Clang 7.

Reply:
Resolved by installing `clang-10` and using `make CC=clang-10` rather than calling `all.sh`.